cclib/method/bpa.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Calculation of Bickelhaupt population analysis (BPA) based on data parsed by cclib."""
+
+import random
+
+import numpy
+
+from cclib.method.population import Population
+
+
+class BPA(Population):

```

Comment:
I would prefer to have a more verbose name here, perhaps just `Bickelhaput`? I know we've been using abbreviations, but I don't think `BPA` is that common this will likely break down in the future for the more advanced methods. @berquist @shivupa thoughts?

Reply:
+1, I also think (separately) that `MPA` should be renamed to Mulliken, etc.

Reply:
Yup, we should definitely rename and leave aliases to the old abbreviations, which we could drop in 2.x.